package.json

```diff
@@ -2,14 +2,15 @@
   "name": "eyevinn-channel-engine",
   "version": "3.3.11",
   "description": "OTT TV Channel Engine",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
I would recommend adding a `"types": dist/index.d.ts` field, that way types can be picked up automatically by IDEs when consumed as a npm package, assuming a d.ts file is generated in the build step.